testing/test_motors.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import context
+import unittest
+import cv2
+import roboplot.core.stepper_motors as stepper_motors
+import roboplot.core.hardware as hardware
+from roboplot.core.gpio.gpio_wrapper import GPIO
+import time
```

Comment:
Good idea!
I've just tweaked it to read that the `context.py` import should be at the top of the *third* group of imports (the `roboplot` ones).